lib/order.rb

```diff
@@ -0,0 +1,17 @@
+class Order
+  attr_reader :total, :as_hash
+
+  def initialize(order, total)
+    @total = total
+    @as_hash = parse order
+  end
+
+  private
+
+  def parse order
+    order.split(', ').inject({}) { |result, item|
```

Comment:
Style/BlockDelimiters: Avoid using `{...}` for multi-line blocks. (https://github.com/bbatsov/ruby-style-guide#single-line-blocks)<br>Style/EachWithObject: Use `each_with_object` instead of `inject`.